pkg/apis/api/componentreport/types.go

```diff
@@ -230,6 +231,9 @@ type ReportTestStats struct {
 
 	// BaseStats may not be present in the response, i.e. new tests regressed because of their pass rate.
 	BaseStats *TestDetailsReleaseStats `json:"base_stats,omitempty"`
+
+	// LastFailure is the last time the regressed test failed.
+	LastFailure *time.Time `json:"last_failure"`
```

Comment:
I wasn't going to mention it but since the linter had issues..  I was curious about consistency here with [TestStatus.LastFailureTime](https://github.com/openshift/sippy/pull/2145/files#diff-5d08c27e4f3c98d1a7feba2a80cb7b626430eea406630aa85857aaea13d72368R127) it's minor and not worth a lot of effort so either way really.

Reply:
Good catch thanks, made them consistent with the shorter form.